flipperdevices/flipperzero-firmware
# [FL-3270] Loader refactoring, part 1

# What's new

- Loader is now thread safe
- Prepared the ground for further refactoring

# Verification 

- Check that applications can be launched from the menu, from the CLI, and from the RPC.
- Make sure that the application or non-parallel-safe cli commands cannot be run if another application is running.

# Checklist (For Reviewer)
